portal/webmap-keyword-search/README.md

```diff
@@ -1,27 +1,31 @@
 # Webmap Keyword Search
 
-Find webmaps within a portal using a keyword.
+Find webmap portal items by using a search term.
 
-![](WebmapKeywordSearch.png)
+![Image of search for webmap](WebmapKeywordSearch.png)
+
+## Use case
+
+Portals can contain many portal items and at times you may wish to query the portal to find what you're looking for. In this example, we search for webmap portal items using a text search.
 
 ## How to use the sample
 
-Input a keyword into the text field and press Enter to search. Click on a result to show the webmap in the map view. Click on the "Find More Results" button to add more results to the list.
+Enter search terms into the search bar. Once the search is complete, a list is populated with the resultant webmaps. Click on a webmap to set it to the map view.  Click on the "Find More Results" button to add more results to the list.
 
 ## How it works
 
-To search for webmaps in a `Portal` matching a keyword:
-
-1. Create a `Portal` and load it
-2. Create `PortalItemQueryParameters`. Set the type to `PortalItem.Type.WEBMAP` and the query to the keyword you want to search
-3. Use `portal.findItemsAsync(params)` to get the first set of matching items.
-4. Get more results with `portal.findItemsAsync(portalQueryResultSet.getNextQueryParameters())`
+1. Create a new `Portal` and load it.
+2. Create new `PortalQueryParameters`. Set the type to `PortalItem.Type.WEBMAP` and add the text you want to search for.
+3. Use `portal.findItemsAsync(params)` to get the first set of matching items (10 by default).
+4. Get more results with `portal.findItemsAsync(portalQueryResultSet.getNextQueryParameters())`.
 
 ## Relevant API
 
-* ArcGISMap
-* MapView
 * Portal
 * PortalItem
 * PortalQueryParameters
 * PortalQueryResultSet
+
+## Tags
+
+keyword, query, search, webmap
```

Comment:
I think all readmes should have an extra newline at the end of the file. It might be there but github is not showing it?